docs/source/quantization.rst

```diff
@@ -84,7 +84,58 @@ PyTorch provides two different modes of quantization: Eager Mode Quantization an
+
+The following table compares the differences between Eager Mode Quantization and FX Graph Mode Quantization:
+
++-----------------+-------------------+-------------------+
+|                 |Eager Mode         |FX Graph           |
+|                 |Quantization       |Mode               |
+|                 |                   |Quantization       |
++-----------------+-------------------+-------------------+
+|Release          |beta               |prototype          |
+|Status           |                   |                   |
++-----------------+-------------------+-------------------+
+|Operator         |Manual             |Automatic          |
+|Fusion           |                   |                   |
++-----------------+-------------------+-------------------+
+|Quant/DeQuant    |Manual             |Automatic          |
+|Placement        |                   |                   |
++-----------------+-------------------+-------------------+
+|Quantizing       |Supported          |Supported          |
+|Modules          |                   |                   |
++-----------------+-------------------+-------------------+
+|Quantizing       |Need Refactor      |Automatic          |
+|Functionals/Torch|(FloatFunctional)  |                   |
+|Ops              |                   |                   |
++-----------------+-------------------+-------------------+
+|Support for      |Limited Support    |Fully              |
+|Customization    |                   |Supported          |
++-----------------+-------------------+-------------------+
+|Quantization Mode|Post Training      |Post Training      |
+|Support          |Quantization:      |Quantization:      |
+|                 |Static, Dynamic,   |Static, Dynamic,   |
+|                 |Weight Only        |Weight Only        |
+|                 |                   |                   |
+|                 |Quantiztion Aware  |Quantiztion Aware  |
+|                 |Training:          |Training:          |
+|                 |Static             |Static             |
++-----------------+-------------------+-------------------+
+|Expect to work on|Yes                |No (We'll integrate|
+|arbitrary model  |                   |FX Graph Mode      |
+|                 |                   |Quantization into  |
+|                 |                   |domain libraries   |
+|                 |                   |like torchvision)  |
++-----------------+-------------------+-------------------+
```

Comment:
I could see people having questions about what `works on an arbitrary model` means, it's possible to interpret this as "quantization will always work on any model without an accuracy drop".  Should we either remove this line or provide some clarifying details?